dowhy/causal_refuter.py

```diff
@@ -176,13 +176,13 @@ def __init__(self, estimated_effect, new_effect, refutation_type):
         self.refutation_result = None
 
     def __str__(self):
-        if self.p_value is None:
+        if self.refutation_result is None:
             return "{0}\nEstimated effect:{1}\nNew effect:{2}\n".format(
                 self.refutation_type, self.estimated_effect, self.new_effect
             )
         else:
             return "{0}\nEstimated effect:{1}\nNew effect:{2}\np value{3}\n".format(
```

Comment:
This needs a change. the output is no longer p-value. You need self.refutation_result['p-value']

Reply:
Done.